src/components/blocks/layout/RichText.tsx

```diff
@@ -5,14 +5,21 @@ import {
 	tinaMarkdownComponents,
 	tinaMarkdownComponentsRichTextTemplate,
 } from '../../fields/TinaMarkdownComponents/TinaMarkdownComponents'
+import { cn } from '../../../lib/utils'
 
 export const RichText = ({ data }: { data: PageBlocksRichText }) => {
-	const { richTextContent } = data
 	return (
-		<div className='max-w-[1360px] mx-auto p-10'>
-			<div className='flex flex-col grow justify-center prose max-w-none prose-headings:my-6 prose-p:my-2 p'>
-				<TinaMarkdown components={tinaMarkdownComponents} content={richTextContent} />
-			</div>
+		<div
+			className={cn(
+				'prose px-4 md:prose-h1:text-6xl prose-h1:text-4xl prose-headings:my-6 prose-p:my-2 max-w-7xl flex flex-col gap-4',
```

Comment:
just need to add `mx-auto` to fix the extra right padding. It is currently restricted due to the max-w-7xl but that's fine bc we don't want it to take up the whole page so we cap it there. The auto horiztonal margin `mx-auto` ensures that the usable maximum width of 7xl (i think ~1280px) is centered too. I

t was not centered without it, hence why it looked like there was too much right padding on the right layout.

Reply:
Thanks @ShawnAviles! I should really learn css layouts this winter break